src/Schema/Directives/ScopeDirective.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+use Nuwave\Lighthouse\Support\Contracts\DefinedDirective;
+
+/**
+ * This directive adds a scope to the builder and supplies the argument value
```

Comment:
Good description, could you move that into the SDL definition below? People are more likely to see that since it is shown in the docs and the autogenerated directives file.